client/extensions/woocommerce/app/dashboard/pre-setup-view.js

```diff
@@ -130,23 +137,25 @@ class PreSetupView extends Component {
 	}
 }
 
-function mapStateToProps( state ) {
-	const site = getSelectedSiteWithFallback( state );
+function mapStateToProps( state, ownProps ) {
 	let name = '';
-	if ( site ) {
-		name = getSiteTitle( state, site.ID );
+	let storeLocation = {};
+	let loading = true;
+
+	if ( ownProps.site ) {
+		name = getSiteTitle( state, ownProps.site.ID );
+		storeLocation = getStoreLocation( state, ownProps.site.ID );
```

Comment:
Agreed! Swapped out in https://github.com/Automattic/wp-calypso/pull/15541/commits/5552a8792688f39b535c77718f94bdaf20514ec9